source/Renderer/shaders/tonemapping.glsl

```diff
@@ -90,9 +101,7 @@ vec3 toneMap(vec3 color)
     // boost exposure as discussed in https://github.com/mrdoob/three.js/pull/19621
     // this factor is based on the exposure correction of Krzysztof Narkowicz in his
     // implemetation of ACES tone mapping
-    color /= 0.6;
-
-    color = toneMapACES_Hill(color);
+    color = toneMapACES_Hill(color / 6.0);
```

Comment:
let's leave this separated, as the division by 0.6 should be emphasised

Reply:
fixed in 0ae406f